pkgs/native_toolchain_c/test/clinker/testfiles/linker/test1.c

```diff
@@ -0,0 +1,6 @@
+#include <stdio.h>
```

Comment:
Don't commit binary files (test.a, test1.o, test2.o).

Add git ignores.

Reply:
But these are test data files - necessary to run the tests.